apps/emqx/src/emqx_os_mon.erl

```diff
@@ -83,12 +82,17 @@ current_sysmem_percent() ->
 %%--------------------------------------------------------------------
 
 init([]) ->
+    %% start os_mon temporarily
+    {ok, _} = application:ensure_all_started(os_mon),
```

Comment:
Q: since this process is started under `emqx_sys_sup` and that in turn is under `emqx` supervision tree, could the application controller enter a dead lock trying to start another application here while itself is starting `emqx`?
Perhaps it would be safer to attempt to start `os_mon` in `handle_continue`?

Reply:
We don't deal with included_applications in emqx_machine.  :(
only use applications and reboot_list.term to get deps digraph, 
So I don't want to add this to `include_applications` to mislead it.